src/info/langs/mod.rs

```diff
@@ -8,27 +8,34 @@ use strum::IntoEnumIterator;
 
 pub mod language;
 
-pub fn get_dominant_language(languages_stat_vec: &[(Language, f64)]) -> Language {
-    languages_stat_vec[0].0
+pub fn get_main_language(loc_by_language: &[(Language, usize)]) -> Language {
+    loc_by_language[0].0
 }
 
-pub fn get_language_statistics(
+pub fn get_loc_by_language_sorted(
```

Comment:
>Better naming for the methods and simplified signatures - up for discussion.

I guess, on this subject, `_loc_` could potentially be confusing to new contributors, since "loc" is frequently an abbreviation for "location". `lines_by_language_sorted` might be slightly more understandable to newcomers.

Reply:
Good point, I added some documentation to clarify the terminology and explain the method's intent.